### PR TITLE
fix: language was not correctly persisted

### PR DESCRIPTION
**Pull request by tpluscode**

### CODE REVIEW

File: api/lib/read-graphs/attribute.ts [hunk 1]
```diff
@@ -14,10 +14,10 @@ handle<AttributeEvents, 'ValueAttributeAdded'>('ValueAttributeAdded', function a
         rdf:predicate <${ev.data.predicate}> .
   `)
 
-  if (ev.data.datatype) {
```
Comment (tpluscode) [inline api/lib/read-graphs/attribute.ts]: The datatype defaults to `xsd:string` and so the `else` was never really hit...

**Comment by martinmaillard** [comment]
That works for now, but we should discuss how we want to handle types (#145)

We will introduce "data type parameters" (e.g. date format"). To me, it would make sense to treat "language" as a parameter of the "string" data type. Even if the backend processes it differently.

**Comment by tpluscode** [comment]
> To me, it would make sense to treat "language" as a parameter of the "string" data type. Even if the backend processes it differently.

This is not allowed by RDF. A typed literal never has language. A literal with language is always implicitly `xsd:string`. Not sure we really want to escape that...

**Comment by martinmaillard** [comment]
Yeah that's what I means by "Even if the backend processes it differently.".

We can show the language as a parameter of "string" in the UI, default to "string" for all properties and then strip the type if the language is defined. I don't think it's a good idea to expose users to the weird choices of the RDF spec. What do you think?

**Pull request merged**